Gemfile

```diff
@@ -146,7 +146,8 @@ group :development, :test do
   gem 'rack-test', require: 'rack/test'
   gem 'rack-vcr'
   gem 'rspec-rails', '~> 3.5'
-  gem 'rubocop', '~> 0.52.1', require: false
```

Comment:
Rubocop upgrade deserves its own PR, but fortunately @thilton-oddball is on it https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/1457

Reply:
@annaswims understood. Had to upgrade to use the auto conversion in `rubocop-rspec`. Pulling that gem out and resetting back to original before this is done. Good catch

Reply:
Removed